lib/server/monalisa-synchronization/MonAlisaSynchronizer.js

```diff
@@ -71,13 +71,22 @@ class MonAlisaSynchronizer {
                 }
             };
 
+            const errors = {};
             for (const monalisaDataPass of dataPasses) {
                 await dataSource.transaction(() => transactionablePipeline(monalisaDataPass))
-                    .catch((error) => this.logger.errorMessage(error.message, {}));
+                    .catch((error) => {
+                        if (!errors[error.message]) {
+                            errors[error.message] = [];
+                        }
+                        errors[error.message].push(monalisaDataPass.name);
+                    });
+            }
+            if (Object.entries(errors).length !== 0) {
+                this.logger.errorMessage(JSON.stringify(errors), {});
             }
             this.logger.debug('Synchronization of Data Passes completed');
         } catch (error) {
-            this.logger.errorMessage(error.message, {});
+            this.logger.errorMessage(`Fatal error preventing synchronization of DataPasses: ${error.message}`, {});
             this.logger.debug('Synchronization of Data Passes failed');
```

Comment:
Why is there this debug if you already logged an error message?

Reply:
removed